rca/people/admin_forms.py

```diff
@@ -6,22 +7,44 @@ class StudentPageAdminForm(WagtailAdminPageForm):
 
     def __init__(self, *args, **kwargs):
         super(StudentPageAdminForm, self).__init__(*args, **kwargs)
-        # Don't allow changing the user account or image collection once set as they are tied to permissions
-        if (
-            self.instance.student_user_image_collection
-            and self.instance.student_user_account
-        ):
-            readonly_fields = [
-                "student_user_image_collection",
-                "student_user_account",
-            ]
-            for f in readonly_fields:
-                # Wrapped in a try except as when a student uses this form, these fields are hidden
-                # See rca.people.models.StudentPage.basic_content_panels and rca.people.utils.PerUserPageMixin
-                try:
-                    self.fields[f].widget.attrs["disabled"] = True
-                except KeyError:
-                    pass
+
+        user = kwargs.get("for_user")
+
+        if user:
```

Comment:
Checking for user here seems to fix the `preview` a draft function for an admin account but not for a student account.